src-docs/src/views/flex/flex_justify_between.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+import { EuiFlexGroup, EuiFlexItem } from '../../../../src/components';
+
+export default () => (
+  <EuiFlexGroup justifyContent="spaceBetween">
+    <EuiFlexItem grow={false}>One here on the left</EuiFlexItem>
```

Comment:
```suggestion
    <EuiFlexItem grow={false}>One here on the left.</EuiFlexItem>
```

All your other statements in these files use periods, as does the line below.